src/components/core/NumberInput/NumberInput.spec.tsx

```diff
@@ -34,23 +34,25 @@ export const numberInputState = async <R extends Renderer = ReactRenderer>({
     },
   );
 
-  await step(
-    "It should not change numberInput value if a non-numeric value is typed",
-    async () => {
-      await userEvent.type(numberInput, "q", {
-        delay: 100,
-      });
+  // ! this test is failing with the Ark UI v1 upgrade, but manually testing it works so it does not reflect real user behavior for some reason
+  // await step(
+  //   "It should not change numberInput value if a non-numeric value is typed",
+  //   async () => {
+  //     await userEvent.type(numberInput, "q", {
+  //       delay: 100,
+  //     });
 
-      await expect(numberInput).not.toHaveValue("q");
-      await expect(numberInput).toHaveValue("");
-    },
-  );
+  //     await expect(numberInput).toHaveValue("");
+  //   },
+  // );
```

Comment:
Hmmm.... yeah I know we talked about this a bit already. Very strange! Definitely cool with just skipping this test for now (especially since the manual testing seems to work as expected).